src/screens/Auth/Login.js

```diff
@@ -39,6 +57,15 @@ export default class Login extends Component {
     return (
       <Wallpaper style={{padding: '10%'}}>
         <Logo />
+        <AmazonSignInButton 
+          onPress={this.loginWithAmazon} 
+          spinning={this.state.amazonWaiting}
+        />
+        <View style={styles.socialDiv}>
+          <Divider />
+          <Text style={styles.divText}>or</Text>
```

Comment:
Lets specify this is an alt for artisans only. We can put that message here or around this divider